src/main/java/redis/clients/jedis/args/ClientAttrOption.java

```diff
@@ -0,0 +1,23 @@
+package redis.clients.jedis.args;
+
+import redis.clients.jedis.util.SafeEncoder;
+
+/**
+ * CLIENT SETINFO command attr option
+ * since redis 7.2
+ */
+public enum ClientAttrOption implements Rawable {
```

Comment:
Sorry but I promise I suggested it before. My browser or internet or something else didn't register it :sweat: 
```suggestion
public enum ClientAttributeOption implements Rawable {
```

Reply:
done